CODE_STYLING.md

```diff
@@ -0,0 +1,180 @@
+## Coding Style Guide
+
+### Common recommendations
+Please, keep source code well-formatted and easy to read!
+
+Use understantable, but not too complicated names.
+Use `TRUE` and `FALSE` for boolean purposes.
+Use predefined constants instead of "magic numbers" (for example, `llAttachToAvatar(ATTACH_NECK);` instead of `llAttachToAvatar(39);`).
+Split different code blocks or steps with spaces.
+Do not make unnecessary complexity.
+Avoid of uncommon hacks (for example, use `llGetListLength(lMyList)` instead of `(lMyList != [])`).
+Avoid of unnecessary memory savings, like loop variables in global scope.
+Feed unicorns, not fire dragons.
+
+### Indentation
+Use spaces, not tabs. Indent size is 4 spaces.
```

Comment:
Tabs are okay though if using the in-world script editor as it converts the tabs to the proper spacing.